docs/DeveloperGuide.md

```diff
@@ -6,9 +6,6 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
 ## Product scope
 ### Target user profile
```

Comment:
There are only 2 user stories. The purpose of a user story is to articulate how a piece of work will deliver a particular value back to the customer. Hence, i'd recommend your team to add in a few more user stories. Maybe 2-3 per person.